netmanager/src/config/urls/deviceMonitoring.js

```diff
@@ -2,7 +2,7 @@ import { stripTrailingSlash } from "../utils";
 
 const BASE_DEVICE_MONITORING_URL = stripTrailingSlash(
   process.env.REACT_APP_BASE_DEVICE_MONITORING_URL ||
-    process.env.REACT_APP_BASE_URL
+    process.env.REACT_APP_BASE_URL_V2
 );
 
```

Comment:
You've changed the device monitoring url to v2 hence the errors in the network monitoring pages. Could you only use this v2 env variable on the api route?
![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/d3d734b7-6c28-430a-8545-174f202a7d1c)